db-acra-migration/README.md

```diff
@@ -0,0 +1,141 @@
+# Migration process, python app, PostgreSQL
+
+Python client application, DB migration, AcraServer, PostgreSQL database.
+
+## 1. Installation
+
+```bash
+curl https://raw.githubusercontent.com/cossacklabs/acra-engineering-demo/master/run.sh | \
+    bash -s -- db-acra-migration
+```
+
+This command downloads a simple Python application that run data migration in a database, Acra Docker containers, PostgreSQL database, sets up the environment, configures python application that run migration process, and provides a list of links for you to try.
+
+## 2. What's inside
+
+**Migration script** is a simple [python console application](https://github.com/cossacklabs/acra-engineering-demo/blob/master/db-acra-migration/python) that works with a database. The application **generate** the data in by copying data from [`users.csv`](https://github.com/cossacklabs/acra-engineering-demo/blob/master/db-acra-migration/python/users.csv) file .The application **migrate** the plaintext data with Acra.
+
+### 2.1 Generate data
+
+```bash
+ docker exec -it db-acra-migration_python_1 python /app/migration.py --generate --port=5432 --host=postgresql
+
+$:
+Data generated successfully!
+```
+
+Call the [`migration.py`](https://github.com/cossacklabs/acra-engineering-demo/blob/master/db-acra-migration/python/migration.py) to create bunch of test data used for migration.
+
+SQL used to store data ([`users.csv`](https://github.com/cossacklabs/acra-engineering-demo/blob/master/db-acra-migration/python/users.csv)) to DB:
+
+```
+COPY users (id,phone_number,ssn,email,firstname,lastname,age) FROM STDIN WITH (FORMAT CSV, HEADER)
+```
+
+Read data directly from DB (password `test`). You should see a bunch of generated test data:
+
+```bash
+docker exec -it db-acra-migration_python_1 psql -h postgresql -U postgres -d test -c "SELECT * from users limit 10";
+
+
+$:
+id | phone_number |    ssn    |             email             | firstname |  lastname  | age 
+----+--------------+-----------+-------------------------------+-----------+------------+-----
+  0 | 8201931087   | 600268041 | josiahharris@lubowitz.info    | Wayne     | Stehr      |  29
+  1 | 5788345157   | 809025969 | carolynbashirian@stanton.com  | Gerardo   | Eichmann   |  50
+  2 | 6281630684   | 762376292 | tinadickinson@farrell.com     | Ralph     | Schoen     |  46
+  3 | 2572577764   | 996643249 | ottilieruecker@konopelski.com | Dylan     | Schiller   |  46
+  4 | 7923022562   | 854151949 | garnettvandervort@denesik.org | Darius    | McKenzie   |  20
+  ....
+```
+
+### 2.3 Update DB
+
+Change DB data types to `bytea` as Acra requires it to store ciphertext. 
```

Comment:
when I read this, it triggers me that miss description about current schema of the database and from what types we migrate now. Let's add one more paragraph at start with the description, that we have data schema like ..... and we generate test data and then will show how to migrate from the not encrypted to the encrypted data with acra.